### PR TITLE
James White

### PR DESCRIPTION
**Pull request by James-E-White**

### CODE REVIEW

**Comment by James-E-White** [comment]
James White 2208 BE

File: lib/car.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Car
+ attr_reader :car, :monthly_payment, :loan_length, :make, :model, :color
+
+  def initialize(car, monthly_payment, loan_length, color = nil)
+    @car = car
+    @make = "Ford"
+    @model = "Mustang"
```
Comment (epintozzi) [inline lib/car.rb:7]: We don't want to hard code these values. What happens if we want to create a Toyota Prius?

File: lib/car.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    @monthly_payment = monthly_payment
+    @loan_length = (36)
```
Comment (epintozzi) [inline lib/car.rb:9]: We don't want to hardcode loan length either. You have a `loan_length` parameter coming in that we can use here instead.

File: lib/car.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    @color = color
+  end
+
+  def total_cost
+    @total_cost = 54000
+    #@monthly_payment * @loan_length
```
Comment (epintozzi) [inline lib/car.rb:15]: The line you have commented out on line 15 is correct. We don't want to hardcode this. We should be calculating the cost of each individual car, not assigning it a static value.

File: lib/dealership.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ def inventory_count
+   @inventory.length
+ end
+
+ def add_car(car)
+   @inventory << car
+ end
+
+ def has_inventory?
+    if @inventory == []
```
Comment (epintozzi) [inline lib/dealership.rb:20]: I think I'd use the `inventory_count` method here instead. Something like:
```ruby
  if inventory_count > 0
    ...
```

**Pull request closed without merging**